src/CLI/clitree/cli-xml/image.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE CLISH_MODULE [
+<!ENTITY START_PORT_ID "1">
+<!ENTITY MAX_PORT_ID "32">
+<!ENTITY START_SUB_PORT_ID "1">
+<!ENTITY MAX_SUB_PORT_ID "4">
+<!ENTITY MAX_MTU "9216">
+]>
+<CLISH_MODULE xmlns="http://www.dellemc.com/sonic/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xi="http://www.w3.org/2001/XInclude" xsi:schemaLocation="http://www.dellemc.com/sonic/XMLSchema http://www.dellemc.com/sonic/XMLSchema/clish.xsd">
+  <VIEW name="enable-view">
+    <COMMAND name="show image-list" help="Show list of images">
+      <ACTION> python $SONIC_CLI_ROOT/sonic-cli-img.py get_openconfig_platform_components image_show.j2 ${__full_line} </ACTION>
+      <xi:include href="include/pipe.xml"/>
+    </COMMAND>
+    <COMMAND name="image" help="image related commands">
+      <ACTION> python $SONIC_CLI_ROOT/sonic-cli-img.py get_openconfig_platform_components image_show.j2 ${__full_line} </ACTION>
+      <xi:include href="include/pipe.xml"/>
+    </COMMAND> 
+    <COMMAND name="image set-default" help="image related commands">
+      <PARAM name="img-path" help="path or url to image" ptype="STRING"> </PARAM>
+      <ACTION> python $SONIC_CLI_ROOT/sonic-cli-img.py get_openconfig_platform_components ${img-path} ${__full_line} </ACTION>
+      <xi:include href="include/pipe.xml"/>
+    </COMMAND>
+    <COMMAND name="image install" help="image related commands">
+      <PARAM name="img-path" help="path or url to image" ptype="STRING"> </PARAM>
+      <ACTION> python $SONIC_CLI_ROOT/sonic-cli-img.py get_openconfig_platform_components ${img-path} ${__full_line} </ACTION>
+      <xi:include href="include/pipe.xml"/>
+    </COMMAND>
+    <COMMAND name="image set-nextboot" help="image related commands">
+      <PARAM name="img-path" help="path or url to image" ptype="STRING"> </PARAM>
+      <ACTION> python $SONIC_CLI_ROOT/sonic-cli-img.py get_openconfig_platform_components ${img-path} ${__full_line} </ACTION>
+      <xi:include href="include/pipe.xml"/>
+    </COMMAND>
+    <COMMAND name="image remove" help="image related commands">
+      <PARAM name="img-path" help="path or url to image" ptype="STRING"> </PARAM>
+      <ACTION> python $SONIC_CLI_ROOT/sonic-cli-img.py get_openconfig_platform_components ${img-path} ${__full_line} </ACTION>
+      <xi:include href="include/pipe.xml"/>
```

Comment:
This pipe support is added by CLI infra. We don't need to have this. This happens during the processing of pre-processed XML file.